### PR TITLE
Add binaries for linux into download map

### PR DESCRIPTION
**Pull request by ccarrascal**
Add Linux binaries to the download map so we don't need to compile from scratch.

### CODE REVIEW

File: scripts/install-binary.js [hunk 1]
```diff
@@ -54,6 +54,10 @@ const DOWNLOAD_MAP = {
     def: 'jq-osx-amd64',
     x64: 'jq-osx-amd64'
   }
```
Comment (davesnx) [inline scripts/install-binary.js:56]: It looks like you missed a comma here

Comment (ccarrascal) [reply scripts/install-binary.js]: Good catch, sorry about that.

**Review by davesnx — APPROVED** [review]

**Comment by huan** [comment]
Thanks for this great improvement!

Could we publish the latest version of the code with this PR to NPM so that we can use it for now?

**Comment by davesnx** [comment]
Sorry about the delay on the publish, it happens manually if you use commitizen commit messages which wasn't the case!

Thanks for the reminder

**Comment by huan** [comment]
Awesome, @davesnx  thank you very muchy for releasing it for us!

**Pull request merged**